model_bakery/baker.py

```diff
@@ -516,20 +535,26 @@ def generate_value(self, field, commit=True):
             generator_attrs.update(filter_rel_attrs(field.name, **self.rel_attrs))
 
         if not commit:
+            old_generator = generator
             generator = getattr(generator, "prepare", generator)
+            if generator is None:
+                generator = old_generator
```

Comment:
Can you add a comment explaining why do we need this new code here?

Reply:
Ow, probably it has something to do with the `ActionGenerator`, right?

Reply:
Yup. Basically, `prepare` defaults to `None` not non-existent, so we need to cope with that

Reply:
No longer relevant now action_generator has gone